### PR TITLE
Assignment 4 Methods

### PR DESCRIPTION
**Pull request by garyanewsome**
Gary A. Newsome, assignment 4 methods

### CODE REVIEW

File: GAN_Assignment4_Methods/src/gan_assignment4_methods/GAN_Assignment4_Methods.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            + "1.) Run the default ASCII word counting program from\n"
+            + "\tweek 1. This will give you the value of the word 'Gary'\n"
+            + "2.) Run the alternate ASCII word counting program\n"
+            + "\tThis alternate will give you the value of\n"
+            + "\ta word of your choosing\n"
+            + "3.) Say 'Hi!'.\n"
+            + "4.) Super top secret.\n"
+            + "5.) Contact the programmer.\n"
+            + "\nPress 0 (zero) to exit.");
+
```
Comment (kavicm) [inline GAN_Assignment4_Methods/src/gan_assignment4_methods/GAN_Assignment4_Methods.java:40]: Very nice program.  I thought it was very well documented.  I had no problem following along.  I ran it in Netbeans and my only suggestion is that it was a little hard to follow when the full menu displayed again.  I like that it keeps running until l choose 0.  Maybe if you just gave the user the option of making another selection from the original menu?  (Note: I am just trying to provide some constructive criticism.  This is way better than mine. :smile: )

Comment (garyanewsome) [inline GAN_Assignment4_Methods/src/gan_assignment4_methods/GAN_Assignment4_Methods.java:40]: @kavicm I actually kind of agree, I had to adjust my Netbeans screen to show a little extra of the output so it was easier to follow and proof. I think it needs a clear screen command, or perhaps a shortened version of the menu to output after the initial run. It can be hard to make it pretty when you are just outputting to the IDE. (No worries, I am hard to offend! ;-)

File: GAN_Assignment4_Methods/src/gan_assignment4_methods/GAN_Assignment4_Methods.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    
+    /**
+     * println method
+     * replaces System.out.print() with println
+     * @param str accepts a string and prints it to the console
+     * with a CR return
+     */
+    public static void println(String str){
+    System.out.println(str);
+    } // end Print Method
```
Comment (ninjapanzer) [inline GAN_Assignment4_Methods/src/gan_assignment4_methods/GAN_Assignment4_Methods.java:98]: Very good job maintaining the original api in these **Helper** methods
